website/netlify.toml

```diff
@@ -0,0 +1,48 @@
+[build]

```

Comment:
do you still want to use netlify to build and deploy the website ?

Reply:
Not really, I'll delete it.